src/modules/roles/domain/exceptions/__tests__/role.exception.spec.ts

```
@@ -0,0 +1,103 @@
+    it('should create exception with correct message', () => {
+      const exception = new CannotRemoveGuestRoleException();
+      expect(exception.message).toBe('Cannot remove last guest role from user.');
+      expect(exception.name).toBe('CannotRemoveGuestRoleException');
+    });
+  });
+
+  describe('RoleExceptions static methods', () => {
+    it('should create RoleNotFoundException via roleNotFound', () => {
+      const exception = RoleExceptions.roleNotFound();
+      expect(exception).toBeInstanceOf(RoleNotFoundException);
+      expect(exception.message).toBe('Role with ID One or more roles not found not found');
```

ESLint

[error] Replace `'Role·with·ID·One·or·more·roles·not·found·not·found'` with `⏎········'Role·with·ID·One·or·more·roles·not·found·not·found',⏎······`

(prettier/prettier)